quick-start/1-build-image-gallery.md

```diff
@@ -69,7 +68,9 @@ $("#images").kendoMobileListView({
     template: "<img src='#: data #'>"
 });
 ```
-* **e**. Run this in the simulator to see how the photos display.
+* **e**. Run this in the simulator to see how the photos display.
+
+**Tip**: In case at some point you see a white screen, most probably JavaScript exception was raised. Please open the developer console by pressing the **F12** key and check for any errors listed under the Console tab. See also [Debug in the Simulator](http://docs.telerik.com/platform/appbuilder/cordova/debugging-your-code/debug-in-simulator).
```

Comment:
> most probably JavaScript exception was raised

most likely a JavaScript exception occurred

> Please open the developer console by pressing the **F12** key and check for any errors listed under the Console tab

F12 is Windows only. You should mention **Command** + **Option** + **I** for OS X users.